Mathlib/SetTheory/Game/Ordinal.lean

```diff
@@ -12,15 +12,14 @@ import Mathlib.SetTheory.Ordinal.NaturalOps
 We define the canonical map `Ordinal → SetTheory.PGame`, where every ordinal is mapped to the
 game whose left set consists of all previous ordinals.
 
-The map to surreals is defined in `Ordinal.toSurreal`.
```

Comment:
I think keeping the definition name would be better, because it becomes a link in the docs